backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -165,6 +184,117 @@ class SchedulingService implements ISchedulingService {
     return schedulingDtos;
   }
 
+  async sendEmailVerificationAfterSchedulingADonation(
+    email: string,
+    schedule: SchedulingDTO,
+  ): Promise<void> {
+    if (!this.emailService) {
+      const errorMessage =
+        "Attempted to call sendEmailVerificationAfterSchedulingADonation but this instance of AuthService does not have an EmailService instance";
+      Logger.error(errorMessage);
+      throw new Error(errorMessage);
+    }
+
+    try {
+      const accountName = await this.userService.getUserByEmail(email);
+      // Proposed drop off info
+      const days = [
+        "Sunday",
+        "Monday",
+        "Tuesday",
+        "Wednesday",
+        "Thursday",
+        "Friday",
+        "Saturday",
+      ];
+      const months = [
+        "January",
+        "February",
+        "March",
+        "April",
+        "May",
+        "June",
+        "July",
+        "August",
+        "September",
+        "October",
+        "November",
+        "December",
+      ];
+      const { startTime } = schedule;
+      const { endTime } = schedule;
+
+      const donationDay: string = days[startTime.getDay()];
+      const donationMonth: string = months[startTime.getMonth()];
```

Comment:
this is going to get the UTC date, we need the local date 

usually this is fine but if it's like 9PM EST Mon, the UTC date will be Tues